css/global.css

```diff
@@ -420,8 +422,7 @@ form {
 }
 /* positioning for body to stop scroll when modal is open */
 .fixed {
-	position: fixed;
-	width: 100%;
+	overflow: hidden;
```

Comment:
worked with group to explain how position fixed was removing entire website out of document flow and sent the user straight up to the top of the website! quick minor change